screens/DoubleSearch.js

```diff
@@ -182,16 +191,24 @@ function DoubleSearch(props) {
      * A.U
      */
     useEffect(() => {
-
-        if (to.name === undefined) {
-            const initialTo = props.navigation.getParam("destinationName", "Destination");
-            setCoordinatesTo(getCoordinates(initialTo));
-            setTo({ name: initialTo });
-        }
+        setCourseScheduleLocation(CourseScheduleLocation);
+        setInitialTo(InitialTo);
+            if (to.name === undefined) {
```

Comment:
can you explain what is going on here?

Reply:
We are getting the props location from course Schedule and Destination from the search bar. I think they are confusing because they are the same as the hooks but they start with a capital letter. I will change their name.

Reply:
It's fixed !!